roles/kubespray-defaults/defaults/main/download.yml

```
@@ -161,6 +161,10 @@ scheduler_plugins_version: "{{ scheduler_plugins_supported_versions[kube_major_v
 
 yq_version: "{{ (yq_checksums['amd64'] | dict2items)[0].key }}"
 
+gateway_api_version: "1.2.1"
+gateway_api_experimental_channel: false
+gateway_api_channel: "{{ 'experimental' if gateway_api_experimental_channel else 'standard' }}"
```

Why not 
```suggestion
gateway_api_channel: "standard" # change to 'experimental' for <here add a ref to upstream doc on what it means)
```
This way we'd avoid having too variable which can diverge (here you can set
```yaml
gateway_api_experimental_channel: false
gateway_api_channel: 'experimental'
```
In the inventory and now we don't know which we should trust.

---

Avoid user typos, but I agree with avoiding too many variables.